go/arrow/cdata/test/test_export_to_cgo.py

```diff
@@ -91,5 +93,22 @@ def test_record_batch(self):
             cgotest.importRecordBatch(self.ptr_schema, self.ptr_array)
 
 
+class TestGoToPython(BaseTestGoPython):
+
+    def test_get_schema(self):
+        with self.assert_pyarrow_memory_released():
+            cgotest.exportSchema(self.ptr_schema)
+
+            sc = pa.Schema._import_from_c(self.ptr_schema)
+            assert sc == self.make_schema()
+    
+    def test_get_batch(self):
+        with self.assert_pyarrow_memory_released():
+            cgotest.exportRecordBatch(self.ptr_schema, self.ptr_array)
+            arrnew = pa.RecordBatch._import_from_c(self.ptr_array, self.ptr_schema)
+            assert arrnew == self.make_batch()
+    
+    
```

Comment:
I would also suggest to add round-tripping tests (for example Python to Go to Python).

Reply:
added round trip tests for schema and record batch that do Python -> Go -> Python and confirm that the resulting schema and record batch are correct